src/scripts/main.js

```diff
@@ -1,3 +1,289 @@
 'use strict';
 
-// write code here
+const body = document.querySelector('body');
+const sortParam = document.querySelector('thead');
+const sortList = document.querySelector('tbody');
+let clickCount = 0;
+
+/* Add event for sorting of table in two directions */
+
+sortParam.addEventListener('click', (e) => {
+  const indexTarget = e.target.cellIndex;
+  let sortedList;
+
+  if (clickCount === 0) {
+    sortedList = sortListAscOrder(indexTarget);
+
+    clickCount = 1;
+  } else {
+    sortedList = sortListDescOrder(indexTarget);
+    clickCount = 0;
+  }
+
+  sortList.append(...sortedList);
+});
+
+function sortListAscOrder(index) {
+  const resultList = [...sortList.rows].sort((a, b) => {
+    const rowA = a.cells[index].innerText.replace('$', '').replace(',', '');
+    const rowB = b.cells[index].innerText.replace('$', '').replace(',', '');
+
+    if (index <= 2) {
```

Comment:
what means this index? if you use it as key of sorting type - make it more obvious